tests/manifold/spherical_manifold_09.cc

```diff
@@ -42,7 +42,7 @@ main()
     for (unsigned int f=0; f<GeometryInfo<dim>::faces_per_cell; ++f)
       if (cell->at_boundary(f))
         {
-          if (std::abs(cell->face(f)->vertex(1).norm()-1.) < 1e-12)
+          if (std::abs(cell->face(f)->vertex(1).norm()-1.) < 1e-1)
```

Comment:
that's pretty drastic, isn't it? why do you need this?

Reply:
The previous threshold was pretty arbitrary. I don't see a reason why we should not just test all the vertices that are located at the outer respectively inner boundary.